com.cubrid.cubridmigration.core/src/com/cubrid/cubridmigration/cubrid/meta/CUBRIDSchemaFetcher.java

```diff
@@ -233,10 +225,12 @@ private void buildCUBRIDTableFKs(Connection conn, Map<String, Table> tables, Sch
 		ResultSet rs = null; //NOPMD
 		Statement stmt = null;
 		try {
-			String sql = "SELECT i.class_name " + "FROM db_index i, db_class c "
-					+ "WHERE i.class_name=c.class_name AND c.is_system_class='NO' "
-					+ "AND c.class_type='CLASS' AND i.is_foreign_key='YES' "
-					+ "GROUP BY i.class_name";
+			String sql = "SELECT i.class_name" 
+					+ " FROM db_index i, db_class c"
+					+ " WHERE i.class_name=c.class_name AND c.is_system_class='NO'"
+					+ " AND c.class_type='CLASS' AND i.is_foreign_key='YES'"
+					+ " GROUP BY i.class_name";
+			
```

Comment:
(trivial) 
I think AS and TOBE is same except indentation, I don't know why you changed.
Litte bit confused.

Reply:
The SQL format is all different, so I modified it to make it easier to see.